app/views/map/index.html.erb

```diff
@@ -0,0 +1,104 @@
+<div id='map' class="map container"> 
```

Comment:
Instead of a single use class you should just use inline css like `style=" height: 1000px; width: 1500px;"`.